api/v1_recommendations_gcp.go

```diff
@@ -31,3 +33,54 @@ func (svc *GcpRecommendationsV1) List() ([]RecommendationV1, error) {
 func (svc *GcpRecommendationsV1) Patch(recommendations RecommendationStateV1) (response RecommendationResponseV1, err error) {
 	return svc.client.Recommendations.patch(GcpRecommendation, recommendations)
 }
+
+// GetReport This is an experimental feature. Returned RecommendationID's are not guaranteed to be correct. Scoped to Lacework Account/Subaccount
+func (svc *GcpRecommendationsV1) GetReport(reportType string) (response []RecommendationV1, err error) {
+	gcpCfg, err := svc.client.Integrations.ListGcpCfg()
+	if err != nil {
+		return []RecommendationV1{}, err
```

Comment:
If you are using named returns you have to use naked returns (somehow)
```suggestion
		return
```

Reply:
Or don't use named returns 😅